ios/Plugin/PluginHelperExtensions.swift

```diff
@@ -41,6 +41,29 @@ internal extension PurchasesPlugin {
         return handleResponse
     }
 
+    func getCompletionBlockHandlerForArrayResponse(
+        _ call: CAPPluginCall,
+        wrapperKey: String? = nil
```

Comment:
Hmm should we make this parameter not optional? I would prefer devs to think about the result of the function, instead of defaulting to `results`.

Reply:
Sounds like a good idea to me! Will make the adjustment 😎

Reply:
Made this change in 9f2be21acde42c922109391487f81623ee46390f and c8b6114a2be236646d601a18e8718e752b97254e. Great idea 😄